knitr/pool-binary-trials/pool-no-pool.Rmd

```diff
@@ -160,33 +216,27 @@ due to numerical arithmetic issues.
 
 Assuming each player's at-bats are independent Bernoulli trials, the
 sampling distribution for each player's number of hits $y_n$ is modeled as
-
-\[
+$$
```

Comment:
This is a minor detail, but generally using `\[ \]` is preferred over `$$ $$`. The later is a pure-TeX shorthand which can cause some issues with LaTeX macros. Not sure it matters for us, but just thought I'd note. [More on Wikibooks](https://en.wikibooks.org/wiki/LaTeX/Mathematics#Mathematics_environments)

Reply:
yes, but only `$$` works in Jupyter notebooks.

Reply:
yes, it would be good to be able to run these in jupyter, either in Python or R.

I used jupyter to update this notebook, so I hit this right off the bat.
awhile back I installed juptext on my machine and I prefer jupyter to RStudio - https://mc-stan.org/users/documentation/case-studies/jupyter_colab_notebooks_2020.html#challenge-putting-the-r-in-jupyter

With juptext and IRkernal, I can run the notebook while keeping it in .Rmd format which means that it's easy to edit via the browser of in emacs.  but that's just me.